src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexInterpreter.cs

```diff
@@ -740,20 +736,26 @@ private bool TryMatchAtCurrentPosition(ReadOnlySpan<char> inputSpan)
                         break;
 
                     case RegexOpcode.Bol:
-                        if (Leftchars() > 0 && inputSpan[runtextpos - 1] != '\n')
                         {
-                            break;
+                            int m1 = runtextpos - 1;
```

Comment:
why `m1` name?

Reply:
minus 1

Reply:
I can rename it subsequently.  We just got a bug report for something I'm fixing and it would conflict with these edits, so I'm merging to avoid those conflicts.